src/edu/ithaca/dragonlab/ckc/io/LearningObjectLinkRecord.java

```diff
@@ -43,12 +61,60 @@ public LearningObjectLinkRecord(){
         return LORLList;
     }
 
+
+    /**
+     * takes a collection of LearningObjects and uses a max possible knowledge estimate to create a list of LOLRs
+     * @param learningObjects
+     * @param maxPossibleKnowledgeEstimate
+     * @return a list of LOLRs
+     */
+    public static List<LearningObjectLinkRecord> createLearningObjectLinkRecords(Collection<LearningObject> learningObjects, double maxPossibleKnowledgeEstimate){
+        List<LearningObjectLinkRecord> lolrList = new ArrayList<LearningObjectLinkRecord>();
+        for(LearningObject learningObject: learningObjects){
+            lolrList.add( new LearningObjectLinkRecord(learningObject.getId(), maxPossibleKnowledgeEstimate));
+        }
+        return lolrList;
+    }
+
+    public static void lolrToJSON(List<LearningObjectLinkRecord> lolrList, String filename)throws IOException{
+        ObjectMapper mapper = new ObjectMapper();
+        mapper.enable(SerializationFeature.INDENT_OUTPUT);
+        mapper.writeValue(new File(filename), lolrList);
+
+    }
+
+
+/**
```

Comment:
What's going on here? IN general, if the code is commented out and being committed, it should at least have a TODO comment letting us know what is happening with it.